sound/soc/codecs/rt715-sdca.h

```diff
@@ -0,0 +1,126 @@
+/* SPDX-License-Identifier: GPL-2.0-only */
+/*
+ * rt715-sdca.h -- RT715 ALSA SoC audio driver header
+ *
+ * Copyright(c) 2019 Realtek Semiconductor Corp.
+*/
+
+#ifndef __RT715_SDCA_H__
+#define __RT715_SDCA_H__
+
+#include <linux/soundwire/sdw.h>
+#include <linux/soundwire/sdw_type.h>
```

Comment:
you still need to forward-declare `struct regmap`, `struct snd_soc_codec`, `struct device` and `#include <linux/workqueue.h>`